src/Chirp.Razor/CheepDTO.cs

```diff
@@ -0,0 +1,6 @@
+﻿public class CheepDTO
+{
+    public string Author { get; set; }
+    public string Text { get; set; }
+    public DateTime Timestamp { get; set; }
```

Comment:
According to the lecture it is advisable that the DTO only consists of predefined types: _"i.e., int, strings, etc. DateTime is not a predefined type."_

Reply:
Then we'll need to refactor it to be the unixtime int and then convert it after retrieval

Reply:
Do we want to just store timesStamp as an int instead, or do we want to convert it twice? Because that feels wasteful.

Reply:
@MathiasJohneck I have changed the datatype, please check, that I did database migration correct...

Reply:
Oops changed the wrong thing, DTO still uses DateTime 😒...

Reply:
It wasn't before, now it is.

when making changes to the database, you add a new migration with:
`dotnet ef migrations add (name relevant to change)`
and then update the database to the new migration with:
`dotnet ef database update`